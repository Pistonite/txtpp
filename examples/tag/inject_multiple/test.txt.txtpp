TXTPP#tag TAG1
-TXTPP#run echo -n "hello tag1"
TXTPP#tag TAG2
-TXTPP#run echo -n "hello tag2"
>>>TAG2TAG1<<<
