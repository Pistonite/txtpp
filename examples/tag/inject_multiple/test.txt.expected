>>>hello tag2hello tag1<<<
